classes/core/PKPRouter.php

```diff
@@ -296,10 +294,10 @@ abstract public function handleAuthorizationFailure(
      * 3) initialization
      * 4) execution
      * 5) client response
-     *
-     * @param $validate whether or not to execute the validation step.
+     * @param array{0:PKPHandler,1:string} $serviceEndpoint
+     * @param bool $validate whether or not to execute the validation step.
      */
-    public function _authorizeInitializeAndCallRequest(callable $serviceEndpoint, PKPRequest $request, array $args, bool $validate = true): void
+    public function _authorizeInitializeAndCallRequest(callable|array $serviceEndpoint, PKPRequest $request, array $args, bool $validate = true): void
```

Comment:
Do we need to have `array` here?
It is maybe similar to call_user_func (s. https://www.php.net/manual/en/function.call-user-func.php) which declaration only contains `collable`, no?

Reply:
Later on we're accessing the context of the callback with an array syntax, which isn't supported by callable (it might not even have a context) :)